assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java

```diff
@@ -988,9 +1006,19 @@ private Assignment mergeAssignment(final String siteId, final Element element, f
 
         // Get an assignment object from the xml
         final Assignment assignmentFromXml = assignmentRepository.fromXML(xml);
+
+        // Remove duplicates from the import
+        String assignmentTitle = assignmentFromXml.getTitle();
+        if ( assignmentTitle == null ) return null;
+        if ( assignmentTitles.contains(assignmentTitle) ) return null;
```

Comment:
this if can be combined and using StringUtils might be a better choice.

Reply:
Turns out there is no such thing but List.contains() handles nulls in both parameters nicely.  

https://docs.oracle.com/javase/8/docs/api/java/util/List.html#contains-java.lang.Object-